src/k8s/cmd/k8s/k8s_status.go

```diff
@@ -59,8 +59,8 @@ func newStatusCmd() *cobra.Command {
 			return f.Print(clusterStatus)
 		},
 	}
-	statusCmd.PersistentFlags().StringVar(&statusCmdOpts.outputFormat, "format", "plain", "Specify in which format the output should be printed. One of plain, json or yaml")
-	statusCmd.PersistentFlags().DurationVar(&statusCmdOpts.timeout, "timeout", 90*time.Second, "The max time to wait for the K8s API server to be ready.")
-	statusCmd.PersistentFlags().BoolVar(&statusCmdOpts.waitReady, "wait-ready", false, "If set, the command will block until at least one cluster node is ready.")
+	statusCmd.PersistentFlags().StringVar(&statusCmdOpts.outputFormat, "format", "plain", "specify in which format the output should be printed. One of plain, json or yaml")
```

Comment:
Is the default also plain here ?

```suggestion
	statusCmd.PersistentFlags().StringVar(&statusCmdOpts.outputFormat, "format", "plain", "specify in which format the output should be printed. One of plain (default), json or yaml")
```

Reply:
We have a PR from @bschimke95 with work on the status command.